### ISSUE TITLE
Gilbrat check isn't correct

### ISSUE THREAD

**Issue by KnathanM** [bp-kelley/descriptastorus#27]
In #22 the following fix was applied to account for scipy changing `gilbrat` to `gibrat`. 

```if dist in ['gilbrat', 'gibrat']:
        # fix change in scikit learn
        if hasattr(dist, 'gilbrat'):
            dist = 'gilbrat'
        else:
            dist = 'gibrat'
```

I think `hasattr(dist, 'gilbrat')` should be `hasattr(st, 'gilbrat')` as the statement will always be false. A string doesn't have the attribute `'gilbrat'`.

**Comment by KnathanM** [comment]
This also fixes #24. scipy 1.9.0 first introduced `gibrat`. descriptastorus 2.6.1 (the current version) always tries to get the `gibrat` attribute from scipy.stats which will fail for any `scipy<1.9`.

**Comment by bp-kelley** [comment]
Thanks for this, for some reason I thought I had fixed it earlier.
